runs/ws_logs.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module handles logs received via websockets.
+"""
+import json
+import time
+
+from .models import ContentChannelRun
+from channels.sessions import channel_session
+from django.http import Http404
+
+
+@channel_session
+def connect(message):
+    # Expected path format: /logs/<run_id>/
+    _, run_id = message['path'].strip('/').split('/')
```

Comment:
Actually never mind the above comment.

In the "On-connect" function we don't have a log record yet.

I guess this is where we would create a Group for this `run_id` so that we can broadcast log messages directly to it, and the js frontend can subscribte to it whenever it loads the status page for this run.